query_frontend/src/provider.rs

```diff
@@ -80,6 +84,42 @@ pub enum Error {
 
 define_result!(Error);
 
+#[derive(Debug, Clone)]
+pub struct ResolvedTable {
```

Comment:
This struct is strange, shouldn't we can get catalog/schema from TableRef directly?